app.py

```diff
@@ -27,6 +30,17 @@
 client = WebClient(token=os.environ["SLACK_TOKEN"])
 client.chat_postMessage(channel=DEPLOYS_CHANNEL_NAME, text="bot started v1.9 240611-1 top")
 
+template = """
+    You're a Software Engineer (Mentor) at OWASP,
+    Your job is to provide help to contributors with a short message.
+    Contributor' Question :{Doubt}
+"""
+
+
+OpenAI_client = OpenAI(api_key=os.getenv("OPENAI_API_KEY"))
```

Comment:
Could you follow [PEP8 naming conventions](https://peps.python.org/pep-0008/#naming-conventions)?

Reply:
Sure !